Makefile

```diff
@@ -0,0 +1,125 @@
+### Virtual Console repacking stuff
+
+## "Option" variables, intended to be overridden from the command line
+# (Or from `config.mk` if you want to persist those)
+
+# Clear this to let ctrtool speak its mind. It is quite talkative.
+VERBOSE_CTRTOOL := >/dev/null
+
+# Paths to the executables (mind that lack of slashes means PATH will be searched instead!)
+CTRTOOL := ctrtool
+MAKEROM := makerom
```

Comment:
Not quite, they can be overridden in the config file without conflicts. (Writing `CTRTOOL := path/to/ctrtool` inside of it will override this default value.)